lib/cli/src/commands/mod.rs

```diff
@@ -84,6 +84,7 @@ pub(crate) trait AsyncCliCommand: Send + Sync {
                     _ = tokio::signal::ctrl_c() => {
                         let term = console::Term::stdout();
                         let _ = term.show_cursor();
+                        std::process::exit(130);
```

Comment:
This only works in Unix. This code will not work in Windows. Also, can we do `128 + SIGINT` instead of `130` ?